de-lib/src/main/java/org/iplantc/de/diskResource/client/presenters/callbacks/DiskResourceMoveCallback.java

```diff
@@ -34,10 +34,15 @@ public void onSuccess(DiskResourceMove result) {
     @Override
     public void onFailure(Integer statusCode, Throwable caught){
         unmaskCaller();
-        DiskResourceErrorAutoBeanFactory factory = GWT.create(DiskResourceErrorAutoBeanFactory.class);
-        AutoBean<ErrorDiskResourceMove> errorBean = AutoBeanCodex.decode(factory, ErrorDiskResourceMove.class, caught.getMessage());
-
-        ErrorHandler.post(errorBean.as(), caught);
+        try {
+            DiskResourceErrorAutoBeanFactory factory = GWT.create(DiskResourceErrorAutoBeanFactory.class);
+            AutoBean<ErrorDiskResourceMove> errorBean =
+                    AutoBeanCodex.decode(factory, ErrorDiskResourceMove.class, caught.getMessage());
+
+            ErrorHandler.post(errorBean.as(), caught);
+        } catch (Exception e) {
+            ErrorHandler.post(caught);
+        }
```

Comment:
@ash3rz @ianmcorvidae I have seen this case happen only for Gateway timeout only happening in Data window.  I am not sure if there are any cases / services that will return non-json response. So for this PR, I just decided to add it places i think we need this logic.